poll/poll.py

```diff
@@ -24,11 +24,14 @@
 from collections import OrderedDict
 import functools
 import json
+import time
 
 from markdown import markdown
 import pkg_resources
 from webob import Response
 
+from courseware.models import StudentModule
+from instructor_task.models import ReportStore
```

Comment:
Please move these imports inside the methods that use them, so that this XBlock can still work from the `xblock-sdk`/`workbench`.

Reply:
Also prefix the imports with `lms.djangoapps..` otherwise they give errors